tests/e2e/test_api.py

```diff
@@ -225,7 +225,7 @@ def test_openapi_endpoint():
     # check application description
     info = payload["info"]
     assert "description" in info, "Service description not provided"
-    assert f"{metadata.SERVICE_NAME} service API specification" in info["description"]
+    assert f"{metadata.SERVICE_NAME} Service API specification" in info["description"]
```

Comment:
its `service` tbh https://github.com/openshift/lightspeed-service/blob/main/docs/openapi.json#L4

Reply:
ok, so it's a problem from the operator branch not having the latest service image but having the latest tests as we've previously discussed. Gotcha.

Reply:
Closing this ticket then, thank you!